ethereumj-core/src/main/java/org/ethereum/sync/SyncManager.java

```diff
@@ -276,6 +296,9 @@ private void produceQueue() {
                     if (wrapper.isNewBlock() && !syncDone) {
                         makeSyncDone();
                     }
+                    if (config.makeDoneByTimeout() >= 0) {
+                        this.lastSyncAction = LocalDateTime.now();
```

Comment:
Do we need this update? During our latest discussion we decided to count that time from sync start up, not from last imported block